src/targets/gpu/kernels/include/migraphx/kernels/ops.hpp

```diff
@@ -118,7 +118,7 @@ struct highest
     template <class T>
     constexpr operator T() const
     {
-        return numeric_max<vec_type<T>>();
+        return numeric_max<vec_type<T>, void>();
```

Comment:
Is there a numeric_min we also have to adjust here?

Reply:
numeric_min is negative of numeric_max. 
It is already defined in type_traits.hpp

Reply:
ok perfect